dotCMS/src/main/java/com/dotmarketing/quartz/QuartzUtils.java

```diff
@@ -775,79 +616,23 @@ public static void startSchedulers () throws SchedulerException {
 		long start = System.currentTimeMillis();
 		Scheduler sched = DotSchedulerFactory.getInstance().getScheduler();
 		sched.start();
-		sched = DotSchedulerFactory.getInstance().getSequentialScheduler();
-		sched.start();
+
 		System.setProperty(WebKeys.DOTCMS_STARTUP_TIME_QUARTZ, String.valueOf(System.currentTimeMillis() - start));
 	}
 	
-	/**
-	 * Temporarily pauses all schedulers from executing future triggers
-	 * @throws SchedulerException 
-	 */
-	public static void startStandardSchedulers () throws SchedulerException {
-		Scheduler sched = DotSchedulerFactory.getInstance().getScheduler();
-		sched.start();
-	}
 
-	/**
-	 * Temporarily pauses all schedulers from executing future triggers
-	 * @throws SchedulerException 
-	 */
-	public static void startSequentialSchedulers () throws SchedulerException {
-		Scheduler sched = DotSchedulerFactory.getInstance().getSequentialScheduler();
-		sched.start();
-	}
 
-    /**
-     * Starts the local quartz scheduler
-     *
-     * @throws SchedulerException
-     */
-    public static void startLocalScheduler () throws SchedulerException {
-        Scheduler localScheduler = DotSchedulerFactory.getInstance().getLocalScheduler();
-        localScheduler.start();
-    }
+	
+	public static boolean isJobRunning(String jobName, String jobGroup) throws SchedulerException{

```

Comment:
@wezell when this gets merge you will notice a new `isJobRunning()` method here: https://github.com/dotCMS/core/pull/19273/commits/704c8c6ebe7d5c3cc85438796c12b4b95b69461b#diff-834b3711e1888aa36bedbb65f457e411R870